test/java/build.gradle

```diff
@@ -24,7 +24,31 @@ task generateSysl(type:Exec) {
   }
 }
 
-compileJava.dependsOn generateSysl
+task generateUserModel(type:Exec) {
+  commandLine "reljam",  "--out", "$tempDirectory", "model", "/relationalmodel", "UserModel"
+
```

Comment:
Can you please delete all lines below until the closing brace of `generateUserModel`? That will make for a way terser code:
```
task generateUserModel(type:Exec) {
  commandLine "reljam",  "--out", "$tempDirectory", "model", "/relationalmodel", "UserModel"
}
```
I've tested it locally, gradle does not capture the output, you still get useful error messages without these lines.

Same for all other `task generate*(type:Exec)`